app/views/reviews/_form.html.erb

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Form</title>
+  </head>
+
+<body>
+  <main>
```

Comment:
why does this form partial also have a <!DOTCTYPE html> , etc.? Since all views are presented in application.html.erb as a <% yield %> portion, aren't these tags redundant? Maybe I'm missing something here...